app/src/main/res/layout/fragment_letter_list.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".unit3.LetterListFragment">
+
+    <androidx.recyclerview.widget.RecyclerView
```

Comment:
we can go just with `androidx.recyclerview.widget.RecyclerView `. No need for FrameLayout